guardrails/cli/hub/install.py

```diff
@@ -171,14 +171,20 @@ def install_hub_module(
     )
     requirements = filter(lambda dep: "extra" not in dep, dependencies)
     for req in requirements:
-        req_info = Stack(*req.split(" "))
-        name = req_info.at(0, "").strip()  # type: ignore
-        versions = req_info.at(1, "").strip("()")  # type: ignore
-        if name:
-            install_spec = name if not versions else f"{name}{versions}"
+        if "git+" in req:
+            install_spec = req.replace(" ", "")
```

Comment:
Might be able to just do this all the time.  Worth testing at least.

Reply:
Hm, it fails the case where versions need to be separated by parentheses 
<img width="1182" alt="Screenshot 2024-05-30 at 12 34 44 PM" src="https://github.com/guardrails-ai/guardrails/assets/7423703/b7317dea-9bdc-489d-b388-59e09088fcf0">